index.android.js

```diff
@@ -9,35 +9,27 @@ import {
   AppRegistry,
   StyleSheet,
   Text,
-  View
+  View,
+  ScrollView,
+  Image
 } from 'react-native';
 
 export default class waterfestival extends Component {
```

Comment:
Nit: Change `waterfestival` to `WaterFestival`? I think it's a good idea to have upper camel-case naming convention for components. This is because you will probably export and import components all over the place and use them like XML. And components offered by libraries follow this naming convention. So for consistency and better legibility, I would recommend using the upper camel-case naming convention.